### PR TITLE
Implement Double Linked List with Methods like (Insert Head, Insert T…

### PR DESCRIPTION
**Pull request by Jimmygm16**
…ail, Remove, Remove_repeated, get_node_index, get_size)

### CODE REVIEW

File: DoubleLinkedList.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    def remove(self, index):
+        if index < 0 or index >= self.length:
+            return None
+        else:
+            self.length -= 1
+            if index == 0:
+                self.head = self.head.next
+                self.head.prev = None
+                return
+            if index == self.length:
```
Comment (L-Zuluaga) [inline DoubleLinkedList.py:42]: Seems like this conditions is affecting your method when trying to remove the last element of the list

**Review by L-Zuluaga — COMMENTED** [review]
I’ve left some comments regarding a few behaviors I noticed during testing. Be sure to always test your code with different edge cases and scenarios to ensure it performs well in all situations.

That aside, the code is well-structured, and you followed good practices by coding in English, and you are familiarized with git commands, which is great 👍🏻.

Overall, I think your implementation was solid, you followed the instructions as required, and you demonstrated a strong understanding of the task. Excellent work 🚀!

**Comment by Jimmygm16** [comment]
> I’ve left some comments regarding a few behaviors I noticed during testing. Be sure to always test your code with different edge cases and scenarios to ensure it performs well in all situations.
> 
> That aside, the code is well-structured, and you followed good practices by coding in English, and you are familiarized with git commands, which is great 👍🏻.
> 
> Overall, I think your implementation was solid, you followed the instructions as required, and you demonstrated a strong understanding of the task. Excellent work 🚀!

You're right Lorenzo, i have to be more

**Comment by Jimmygm16** [comment]
> I’ve left some comments regarding a few behaviors I noticed during testing. Be sure to always test your code with different edge cases and scenarios to ensure it performs well in all situations.
> 
> That aside, the code is well-structured, and you followed good practices by coding in English, and you are familiarized with git commands, which is great 👍🏻.
> 
> Overall, I think your implementation was solid, you followed the instructions as required, and you demonstrated a strong understanding of the task. Excellent work 🚀!

You're right Lorenzo, I should be test my functions better. I will take it into account. Thanks for the feedback and the time!
